src/main/kotlin/no/nav/syfo/utland/UtenlandskSykmeldingService.kt

```diff
@@ -50,11 +50,12 @@ class UtenlandskSykmeldingService(
         }
     }
 
-    fun behandlesISykDig(tildeltEnhetsnr: String?, fnr: String, loggingMeta: LoggingMeta): Boolean {
-        return if (cluster == "dev-gcp") {
+    fun behandlesISykDig(tildeltEnhetsnr: String?, oppgave: OppgaveResultat?, loggingMeta: LoggingMeta): Boolean {
```

Comment:
Er det ikke bedre å la nullsjekken være der den var så du slipper å gjøre den to ganger? Og heller bare sende inn oppgaveid-en i denne metoden? :)

Reply:
Jo det er sant